pkgs/applications/blockchains/feather-wallet/install_fix.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 481874d..6dbe2b2 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -302,5 +302,5 @@ if(APPLE)
+ endif()
+ 
+ install(TARGETS feather
+-        DESTINATION ${CMAKE_INSTALL_PREFIX}
++        DESTINATION ${CMAKE_INSTALL_PREFIX}/bin
```

Comment:
Is it possible to set INSTALL_PREFIX via cmakeFlags?

Reply:
I think the `CMAKE_INSTALL_PREFIX` is fine, the problem is that the executable was being installed at the root, not in the `${CMAKE_INSTALL_PREFIX}/bin` directory. I should probably report this to upstream.